samples/hello++/main.cpp

```diff
@@ -0,0 +1,40 @@
+#include <xboxrt/debug.h>
+#include <pbkit/pbkit.h>
+#include <hal/xbox.h>
+#include "stdio.h"
+
+class hello{
+private:
+  int hiddenValue = 0;
+public:
+  hello() {hiddenValue = 5;};
+  hello(int v) : hiddenValue(v) {};
+  int getValue() { return hiddenValue; };
+};
+
+int main() {
+  switch(pb_init())
+  {
+  case 0: break;
+  default:
+    XSleep(2000);
+    XReboot();
+    return -1;
+  }
```

Comment:
I always cringe with this init sequence; I suggest:

```
  int ret = pb_init();
  if (ret != 0) {
    XSleep(2000);
    XReboot();
    return -1;
  }
```

*And possibly a PR to change it in our other code, too*

Reply:
Taken care of in this instance.
I'll file an issue about the other samples.